yarn.lock

```diff
@@ -6602,7 +6602,7 @@
   resolved "https://registry.yarnpkg.com/@ovh-ux/translate-async-loader/-/translate-async-loader-1.0.8.tgz#7cbdc3dfed1042f511206ff6bb9f14c33b8406ba"
   integrity sha512-8yEhPeRTSxtEO9ODXRYZw6pzRQiV5ULCTi8mIzt39m/b9N2TcIF4ILTPsaa8xpE4npHKa/iYgL9z0HJUFi2LGQ==
 
-"@ovh-ux/ui-kit@^6.5.1":
+"@ovh-ux/ui-kit@^6.5.0", "@ovh-ux/ui-kit@^6.5.1":
```

Comment:
I don't know if it's normal . Why all project doesn't have only one version of ui-kit ?